autoload/utils/exec/job.vim

```diff
@@ -44,6 +44,15 @@ function! s:vimClose(channel) abort
     call s:createQuickFix()
     call s:closeBuffer()
 
+    if has_key(s:cmake4vim_job, 'job')
+        if job_info(s:cmake4vim_job['job'])['exitval'] == 0
+            call utils#common#Warning('Success!')
```

Comment:
Could you maybe also print the command line?

Reply:
Hmmm, I'm not sure I understand, what should I print? `s:cmake4vim_job['cmd']`? I didn't print that because it's already in `w:quickfix_title`

Reply:
Yes it is that I mean. 
Could you please add the `s:cmake4vim_job['cmd']` for `system` executor.